applications/GeoMechanicsApplication/custom_workflows/time_loop_executor.hpp

```diff
@@ -63,6 +64,7 @@ class TimeLoopExecutor : public TimeLoopExecutorInterface
         mStrategyWrapper->SaveTotalDisplacementFieldAtStartOfTimeLoop();
         std::vector<TimeStepEndState> result;
         TimeStepEndState              NewEndState = EndState;
+        ScopedOutputFileAccess        limit_output_file_access_to_this_scope{*mStrategyWrapper};
```

Comment:
I don't think that is necessary. For instance, one of our colleagues was not aware of the name of this idiom (RAII), but he perfectly understood how this class works. It would, however, be a good candidate for our internal knowledge sharing program.